src/bundle_add.c

```diff
@@ -533,19 +524,7 @@ enum swupd_code main_bundle_add()
 enum swupd_code bundle_add_main(int argc, char **argv)
 {
 	int ret;
-	const int steps_in_bundleadd = 7;
-
-	/*
-	 * Steps for bundle-add:
-	 *
-	 *  1) load_manifests
-	 *  2) consolidate_files
-	 *  3) check_disk_space_availability
-	 *  4) download_packs
-	 *  5) download_fullfiles
-	 *  6) install_files
-	 *  7) progress_set_step
-	 */
+	const int steps_in_bundleadd = 8;
```

Comment:
Maintaining the number of steps is hard anyway. My suggestion is actually to remove the total number of steps because the installer isn't using it. I'm just waiting this patch to get in to suggest the other change.

Reply:
the initial idea of having the total number of steps was because that is the only way to be able to show real progress, otherwise you can only show the progress of the current step but it is impossible to know the progress of the whole process. 

So my idea originally was that the installer (or perhaps a GUI tool for swupd in the future) was going to show an overall progress of the whole swupd operation, while the percentages in a step could be used to show progress of the current action. That way you could have something like:
```
$swupd bundle-add BUNDLE

|------>           |   Overall bundle-add progress
|----------------->|     - loading manifests
|----------------->|     - downloading packs
|----------->      |     - installing files
```

At the moment the number of steps have not been used by any gui, but I don't think we should get rid of it in case at some point some tool wants to show progress like that.

Reply:
ok. I won't remove that then. And I'll add comments for all the steps back.